.github/workflows/docs.yml

```
@@ -40,6 +40,8 @@ jobs:
         run: make docs-html
 
       - uses: errata-ai/vale-action@reviewdog
+        with:
+          files: docs/**/*.md
```

Fix path. However, the argument to files should include multiple subdirectories. You need need to add `docs/source/**/*.md` and `docs/source/**/**/*.md` and so on.

```suggestion
          files: docs/source/*.md
```

---

Also to save iterations, check the Documentation build in GitHub Actions to see what it is doing.

For example: https://github.com/plone/volto/actions/runs/4262031608/jobs/7417049297#step:9:30

---

> Fix path. However, the argument to files should include multiple subdirectories. You need need to add docs/source/**/*.md and docs/source/**/**/*.md and so on.

thaanks I was looking at how to fix it.... didn't ask you since i didn't want to bother you😅